dataland-frontend/tests/e2e/specs/infrastructure/VerifyHeaders.ts

```diff
@@ -37,7 +37,7 @@ describe("As a developer, I want to ensure that security relevant headers are se
   describeIf(
     "Check CSP headers in the CI/CD environment",
     {
-      executionEnvironments: ["development", "preview"],
+      executionEnvironments: ["development", "development_2", "preview"],
```

Comment:
I think in the definition of the cypress environments in the CD or CI file we could filter (if that is easily possible in a yaml file) for the word 'development' in the environment name and then just return that as the cypress environment name. What do you think of that?

Reply:
This would be a good idea if all development environments always run the same tests.
But I just noticed that there exist tests, which run e.g. on dev1 and dev2, but not on devLocal, and vice versa.
So in my optinion, we can keep it like it is now.
Furthermore I don't think that many new executionEnvironments could possibly be added in the future.  So we shouldn't get to the point that you lose the overview here.

Reply:
Ist that ok for you too?

Reply:
Yes.